src/components/Table/Filters.tsx

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import { Input, Icon } from "antd";
+import { Button } from "components/Button";
+import { TreeDataEntry } from "components/TreeSelect";
+import { CheckboxGroup } from "components/Checkbox";
+import styled from "@emotion/styled";
+
+export interface InputFilterProps {
+  dataCy?: string;
+  placeholder: string;
+  value: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  updateUrlParam: () => void;
+  resetUrlParam: () => void;
+}
+
+export const InputFilter: React.FC<InputFilterProps> = ({
+  placeholder,
+  value,
+  onChange,
+  dataCy,
+  updateUrlParam,
+  resetUrlParam,
+}) => (
+  <FilterWrapper>
+    <Input
+      aria-label="chicken"
```

Comment:
we should say something more descriptive here

Reply:
lol typo